defender/utils.py

```diff
@@ -202,8 +202,8 @@ def unblock_username(username, pipe=None):
         pipe = REDIS_SERVER.pipeline()
         do_commit = True
     if username:
-        pipe.delete(get_username_attempt_cache_key(username))
-        pipe.delete(get_username_blocked_cache_key(username))
+        pipe.delete(get_username_attempt_cache_key(username.lower()))
+        pipe.delete(get_username_blocked_cache_key(username.lower()))
```

Comment:
I wasn't sure why these lines where here twice?   

Does it make sense to leave them and then add two more that do the same with the username.lower()?